src/controllers/userController.js

```diff
@@ -40,6 +43,8 @@ const login = async(req,res) => {
 
     const userInfo = user.toObject();
     delete userInfo.password;
+    delete userInfo._id;
```

Comment:
instead of deleting we can just select what to return. IE:
    const { email: userEmail, firstName, lastName } = user;